src/Microsoft.Azure.WebJobs.Host/Utility.cs

```diff
@@ -50,5 +50,16 @@ internal static string GetFunctionShortName(MethodInfo methodInfo)
 
             return attribute?.ConfigurationSection ?? GetExtensionAliasFromTypeName(type.Name);
         }
+
+        internal static int GetProcessorCount()
```

Comment:
Shouldn't need this copy - the tests have internals visible to the Storage extension. If they don't, you can add it https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host.Storage/Properties/AssemblyInfo.cs. You want the tests to be calling the same helper that the product code is